### ISSUE TITLE
LinearLR can't be imported this way anymore.

### ISSUE THREAD

**Issue by abhimanyuchadha96** [pytorch/vision#4467]
https://github.com/pytorch/vision/blob/972ca657416c5896ba6e0f5fe7c0d0f3e99e1087/references/detection/engine.py#L24

**Comment by oke-aditya** [comment]
Hi !
I think LinearLR is not available in pytorch 1.9.1 but available over PyTorch master

PyTorch 1.9.1 docs
https://pytorch.org/docs/1.9.1/optim.html#how-to-adjust-learning-rate

these do not have it.

Master docs https://pytorch.org/docs/master/optim.html
You can find LinearLR here.
https://pytorch.org/docs/master/generated/torch.optim.lr_scheduler.LinearLR.html#torch.optim.lr_scheduler.LinearLR

**Comment by abhimanyuchadha96** [comment]
I tried downgrading to pytorch 1.8 and the results were the same.

**Comment by oke-aditya** [comment]
Well downgrading to torch 1.8 will not help

![image](https://user-images.githubusercontent.com/47158509/134411743-76238f50-bb50-421d-b687-dceb8a4a18aa.png)

As you see LinearLR isn't part of PyTorch 1.8

**Comment by oke-aditya** [comment]
You would need to use pytorch nightly / probably wait for PyTorch 1.10

**Comment by oke-aditya** [comment]
Here is a screenshot from my Ubuntu machine. It shows that LinearLR can be used in torch 1.10 nightly

![image](https://user-images.githubusercontent.com/47158509/134412054-791678a0-2f68-41cd-9bad-da9c97c5e7fe.png)

**Comment by datumbox** [comment]
As @oke-aditya said, LinearLR is a brand new scheduler that exists only on the latest nighly version of Pytorch. Downgrading won't help. You need to upgrade to the very latest available non-stable nighly version.
